python/taichi/lang/__init__.py

```diff
@@ -360,7 +374,7 @@ def wrapped(*test_args, **test_kwargs):
             ip = kwargs.get('default_ip', ti.i32)
             if fp == ti.f64 or ip == ti.i64:
                 can_run_on.register(
-                    lambda arch: is_supported(arch, extension.data64))
+                    lambda arch: is_extension_supported(arch, extension.data64))
 
             for arch in ti.supported_archs():
```

Comment:
In this for loop we still have 2 `print`, I don't know whether do we need to convert them to `ti.info`